README.md

```diff
@@ -1,79 +1,58 @@
 
-To compile the binary, run:
+To get the code and compile the binary, run:
 
 ```bash
-mkdir $GOPATH/src
-mv the/sources/dir/docker-machine-driver-g5k $GOPATH/src
-go get -d -u github.com/docker/machine
-cd $GOPATH/src/docker-machine-driver-g5k
-go install
+go get -u github.com/Spirals-Team/docker-machine-driver-g5k
 ```
 
 Then, either put the driver in a directory filled in your PATH environment variable, or run:
 
 ```bash
-export PATH=/my/driver/dir/:$PATH
+export PATH=$PATH:$GOPATH/bin
 ```
 
-Be sure that your SSH public keys file is `$HOME/.ssh/id_rsa.pub`
+## How to use
 
-### Grid5000
-You should have an `authorized_keys` file in your SSH directory. Put it on your public directory, on each site you wish to create a machine on.
+### VPN
+You need to be connected to the Grid5000 VPN to create and access your Docker node.
+Dont forget to configure your DNS or use OpenVPN DNS auto-configuration.
+Please follow the instructions on the [Grid5000 Wiki](https://www.grid5000.fr/mediawiki/index.php/VPN).
 
-## How to use
 ### Options
-The driver will need a few options to create a machine. Here is a list of options:
-
-|       Option      |  Description     |  Default value   | Required  |
-|-------------------|------------------|------------------|-----------|
-| `--g5k-username`  | User's account   |                  | Yes       |
-| `--g5k-passwd`    | User's passwd    |                  | Yes       |
-| `--g5k-site`      | Site's location of the machine |    | Yes       |
-| `--g5k-walltime`  | Timelife of the machine | "1:00:00" | No        |
+The driver need a few options to create a machine. Here is a list of options:
+
+|          Option          |              Description              |     Default value     |  Required  |
+|--------------------------|---------------------------------------|-----------------------|------------|
+| `--g5k-username`         | Your Grid5000 account username        |                       | Yes        |
+| `--g5k-password`         | Your Grid5000 account password        |                       | Yes        |
+| `--g5k-site`             | Site to reserve the resources on      |                       | Yes        |
+| `--g5k-walltime`         | Timelife of the machine               | "1:00:00"             | No         |
+| `--g5k-ssh-private-key`  | Path of your ssh private key          | "~/.ssh/id_rsa"       | No         |
+| `--g5k-ssh-public-key`   | Path of your ssh public key           | "< private-key >.pub" | No         |
+| `--g5k-image`            | Name of the image to deploy           | "jessie-x64-min"      | No         |
 
 ### Example
-An example :
+An example of machine creation :
 
 ```bash
 docker-machine create -d g5k \
 --g5k-username user \
---g5k-passwd ******** \
+--g5k-password ******** \
 --g5k-site lille \
 --g5k-walltime 2:45:00
+--g5k-ssh-private-key ~/.ssh/g5k-key
 ```
```

Comment:
It can also be useful to add an option for directly selecting a node (and/or to add a filter for hardware requirements (for example, on the processor name, if the node has power measurement capabilities, etc.)).